backend/src/test/java/serviceTests/ItemServiceTest.java

```diff
@@ -0,0 +1,719 @@
+        item1.setDescription("Description1");
+        item1.setImageSrc("ImageSrc1");
+        item1.setCategory(category);
+
+        Item item2 = new Item();
+        item1.setId(2L);
+        item1.setName("Item2");
+        item1.setDescription("Description2");
+        item1.setImageSrc("ImageSrc2");
+        item1.setCategory(category);
+
+        Item item3 = new Item();
+        item1.setId(3L);
+        item1.setName("Item3");
+        item1.setDescription("Description3");
+        item1.setImageSrc("ImageSrc3");
+        item1.setCategory(category);
+
+        itemList.add(item1);
+        itemList.add(item2);
+        itemList.add(item3);
+
+        List<ItemCreateDTO> itemCreateDTOList = new ArrayList<>();
+
+        ItemCreateDTO itemCreateDTO1 = new ItemCreateDTO();
+        itemCreateDTO1.setName("Item1");
+        itemCreateDTO1.setDescription("Description1");
+        itemCreateDTO1.setCategoryId(1L);
+        itemCreateDTO1.setImageSrc("ImgSrc1");
+
+        ItemCreateDTO itemCreateDTO2 = new ItemCreateDTO();
+        itemCreateDTO2.setName("Item2");
+        itemCreateDTO2.setDescription("Description2");
+        itemCreateDTO2.setCategoryId(1L);
+        itemCreateDTO2.setImageSrc("ImgSrc2");
+
+        ItemCreateDTO itemCreateDTO3 = new ItemCreateDTO();
+        itemCreateDTO3.setName("Item3");
+        itemCreateDTO3.setDescription("Description3");
+        itemCreateDTO3.setCategoryId(1L);
+        itemCreateDTO3.setImageSrc("ImgSrc3");
+
+        itemCreateDTOList.add(itemCreateDTO1);
+        itemCreateDTOList.add(itemCreateDTO2);
+        itemCreateDTOList.add(itemCreateDTO3);
+
+        List<ItemDTO> expectedItemDTOList = new ArrayList<>();
+
+        ItemDTO itemDTO1 = new ItemDTO();
+        itemDTO1.setId(1L);
+        itemDTO1.setName("Item1");
+        itemDTO1.setDescription("Description1");
+        itemDTO1.setCategoryId(1L);
+        itemDTO1.setCategoryName("Category");
+        itemDTO1.setImageSrc("ImgSrc1");
+
+        ItemDTO itemDTO2 = new ItemDTO();
+        itemDTO2.setId(2L);
+        itemDTO2.setName("Item2");
+        itemDTO2.setDescription("Description2");
+        itemDTO2.setCategoryId(1L);
+        itemDTO2.setCategoryName("Category");
+        itemDTO2.setImageSrc("ImgSrc2");
+
+        ItemDTO itemDTO3 = new ItemDTO();
+        itemDTO3.setId(3L);
+        itemDTO3.setName("Item3");
+        itemDTO3.setDescription("Description3");
+        itemDTO3.setCategoryId(1L);
+        itemDTO3.setCategoryName("Category");
+        itemDTO3.setImageSrc("ImgSrc3");
+
+        expectedItemDTOList.add(itemDTO1);
+        expectedItemDTOList.add(itemDTO2);
+        expectedItemDTOList.add(itemDTO3);
+
+//        when(itemMapper.toEntity(itemCreateDTOList)).thenReturn(itemList);
+        when(itemRepository.saveAll(any())).thenReturn(itemList);
+        when(itemMapper.toDto(itemList)).thenReturn(expectedItemDTOList);
+
+        List<ItemDTO> result = itemService.uploadItemsFromCsv(csvFile);
```

Comment:
you can run tests with "Coverage" and you'll see you missed case when `uploadItemsFromCsv` throws exception